.circleci/workflows.yml

```diff
@@ -1181,6 +1182,24 @@ jobs:
           when: always
       - store_test_results:
           path: ~/test-results
+  pull_requests_ce:
+    executor:
+      name: openjdk
+      class: large
+      version: "21.0.6"
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - checkout
+      - restore-maven-job-cache:
+          jobName: pull_requests
+      - mvn_git_commit_id
+      - run:
+          name: "Maven Package and install without tests"
+          command: |
+            mvn -s /tmp/.gravitee.settings.xml -P gio-artifactory-snapshot clean install -DskipTests
```

Comment:
the with-mock has to be keep IMO so the bundle will contains extra plugin for the integration test phase